config/certificates/chain-cas-utt.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN CERTIFICATE-----
+MIIHHzCCBgegAwIBAgIQBA8Yo15aY/oEcpojdkcyBjANBgkqhkiG9w0BAQsFADBkMQswCQYDVQQGEwJOTDEWMBQGA1UECBMNTm9vcmQtSG9sbGFuZDESMBAGA1UEBxMJQW1zdGVyZGFtMQ8wDQYDVQQKEwZURVJFTkExGDAWBgNVBAMTD1RFUkVOQSBTU0wgQ0EgMzAeFw0xODEyMTQwMDAwMDBaFw0yMTAzMTgwMDAwMDBaMG4xCzAJBgNVBAYTAkZSMQ8wDQYDVQQHEwZUcm95ZXMxLTArBgNVBAoMJFVuaXZlcnNpdMOpIGRlIFRlY2hub2xvZ2llIGRlIFRyb3llczEMMAoGA1UECxMDVVRUMREwDwYDVQQDDAgqLnV0dC5mcjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKBDz5C+l9s67HOc0E4xMX+gt8DnLrDi1IDNyWOXLzLd/OUqOmJe7wyUfmPbeLakLpdLCkMIy9hxc/GsF0lI7P5EJZUs3quf5d/FCnsxzdLHBN3I9ZITz5tdqRFuvMXNAN2ZP92efwpfDybD+at7tDlAyxNteHn6IImi30+zwLsrpYxIXJAeHil7yR189HYlydoQPmYLMMNKc/kmYq6u6rKnllGdTrGNl8adM5MxBp5npgcc+xGY9Qjv2bHsQk3GR/Fv238hw2xHyFc/9RGsml1ZNMxG2iPu1fx49VE/kAhQiz2JY0BYlGVHozUiKmDtI3r11FxkpE/r1FOtIJPmOMUCAwEAAaOCA8EwggO9MB8GA1UdIwQYMBaAFGf9iCAUJ5jHCdIlGbvpURFjdVBiMB0GA1UdDgQWBBRxu5w1EajPuSiHjxsyRtjjOU+UMzAbBgNVHREEFDASgggqLnV0dC5mcoIGdXR0LmZyMA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwawYDVR0fBGQwYjAvoC2gK4YpaHR0cDovL2NybDMuZGlnaWNlcnQuY29tL1RFUkVOQVNTTENBMy5jcmwwL6AtoCuGKWh0dHA6Ly9jcmw0LmRpZ2ljZXJ0LmNvbS9URVJFTkFTU0xDQTMuY3JsMEwGA1UdIARFMEMwNwYJYIZIAYb9bAEBMCowKAYIKwYBBQUHAgEWHGh0dHBzOi8vd3d3LmRpZ2ljZXJ0LmNvbS9DUFMwCAYGZ4EMAQICMG4GCCsGAQUFBwEBBGIwYDAkBggrBgEFBQcwAYYYaHR0cDovL29jc3AuZGlnaWNlcnQuY29tMDgGCCsGAQUFBzAChixodHRwOi8vY2FjZXJ0cy5kaWdpY2VydC5jb20vVEVSRU5BU1NMQ0EzLmNydDAMBgNVHRMBAf8EAjAAMIIB9AYKKwYBBAHWeQIEAgSCAeQEggHgAd4AdQC72d+8H4pxtZOUI5eqkntHOFeVCqtS6BqQlmQ2jh7RhQAAAWesXbUwAAAEAwBGMEQCIF62E0h5avjNI7o3npn2TwXsUmJpOFZtKrcGoEmFO2G9AiBjpc/aWN+VY1EL5rlY3oQstZjkvlY9QhpftZTzBnTjpAB1AId1v+dZfPiMQ5lfvfNu/1aNR1Y2/0q1YMG06v9eoIMPAAABZ6xdtgYAAAQDAEYwRAIgG1nmi29DSphWjVTh2cwHiAX8Ps2Wolsis/ePtSZfGJACIBf+Htom87gVps+bzXgJEJn8IMj4cXJYvO5nPoCqGQTSAHUAb1N2rDHwMRnYmQCkURX/dxUcEdkCwQApBo2yCJo32RMAAAFnrF22kQAABAMARjBEAiAxQPinBrCpK0Ou1eMjqXqxHWC3BfDUNnW5C0OaNo9QWwIgJOXK0ddZi2lXg2nMhdklz1GgYva6bP5GDWHn3Dz+4VEAdwBVgdTCFpA2AUrqC5tXPFPwwOQ4eHAlCBcvo6odBxPTDAAAAWesXbcEAAAEAwBIMEYCIQCgJ1LMrQgisRCPFzc+/oH6Bf9/dyeKbYp38GGsv5TBogIhAJzOFLFtLNhtFXMhG61awsFmVRtHeaGcBuo+lLTIsE1oMA0GCSqGSIb3DQEBCwUAA4IBAQB2pU9qqBYzYoMgJ1sbJ2QDSV2pRGHVBr/sIkkorbm+i0XlZa0pB6XbPwF6X05mwIV5QyJ/ZfhN4nyHXSnMFdpCWZQU/vTKlRCMxjWdfwe76ToGKVOh+nTB3XpPxzy0+nmDOANivo5zNeXqYJPvlq7LmmoZIMGNC0uScK4o7yzl0bZyKpD4nW6xM1BZiiHzfcFJPRAEgGpv2+rneeoXYQtKeupLiT+lIgF/rQ5Az9zSebrBhT55LHEFIKj6Cnpd4Mm54Ibkq7XSc2uwEjnPSisnZngJSCW/mpf9RYcFavJzF+6v8xUaDZe2JLFut12zXHfua8WZyCvorfCHJiPwXImY
```

Comment:
do we really want to have certificate clear on repo ???

Reply:
In my humble opinion, yes we do. This is a public certificate used to build & validate a PKIX chain of trust, hence the lack of a private key markup.
Indeed, at the best of my knowledge, we do not posses something to reliably store it in a way that prevents malicious threats, such as a HSM or anything else that provide sufficient tamper-resistance.
Therefore, storing it on a public repository that is both auditable by multiple parties and convenient to modify in the event of such required action, does not seem to qualify as a mismanagement of any sort.

Reply:
@BeauneNuits @arnaud9145 @larueli we could also use Symfony secrets if need be, same goes for potential private keys. 

See https://symfony.com/doc/current/configuration/secrets.html

Reply:
@MrGabigoo Although we probably _could_ , do we actually _need_ to do so ? Seems a tad overdone to store public certificates.
As for the private keys, I reckon the current use of both Ansible Vault and K8S secrets fulfills our needs.

Reply:
@BeauneNuits fair point! In addition, using Ansible Vault and/or k8s secrets is more compliant with the 12-factor principle of using environment defined configuration :)

Reply:
Keep cool, don't worry ;) it's only a public certificate, accessible by going on cas.utt.fr, used to validate the communication with the cas. Plus, anyone can check and compare the certificate stored here (and used by the gateway) with certificate on cas.utt.fr.

Yes, multiples solutions exists to store secrets, but k8s secrets management suits our needs for now.